pyfar/dsp/dsp.py

```diff
@@ -1498,3 +1498,144 @@ def decibel(signal, domain='freq', log_prefix=None, log_reference=1,
         return log_prefix * np.log10(np.abs(data) / log_reference), log_prefix
     else:
         return log_prefix * np.log10(np.abs(data) / log_reference)
+
+
+def energy(signal, keepdims=False):
+    r"""
+    Compute the energy of a signal.
+
+    According to the Parseval's theorem [#]_ the energy is
+
+    .. math::
+
+        \sum_{n=0}^{N-1}|x[n]|^2=\frac{1}{N}\sum_{k=0}^{N-1}|X[k]|^2,
+
+    The function calculates the energy from the time data for each channel
+    separately.
+
+    Parameters
+    ----------
+    signal : Signal
+        The signal to compute the energy from.
+    keepdims : bool, optional
+        If ``False``, the shape of the returned array equals ``signal.cshape``.
+        If ``True``, an axis will be appended, so it's broadcastable to the
+        shape of ``signal.time`` and ``signal.freq``.
+        The default is ``False``.
```

Comment:
1. This is a bit misleading in my opinion. Keeping the dimensions to me sounds like "we maintain the dimensionality of the signal for which we calculate the energy" so, naturally I would expect a True here.
2. What's do we actually need this for?